tests/AssetManagerTest/Resolver/AggregateResolverTest.php

```diff
@@ -69,5 +74,21 @@ public function testCollect()
         $this->assertContains('three', $collection);
 
         $this->assertCount(3, $collection);
+
+        /* Tests for interfaces that _don't_ implement the `collect` method. */
+        $resolver    = new AggregateResolver();
```

Comment:
This is a specific scenario and needs a specific test case with a specific name.